src/store/account-details/account-details.thunks.ts

```diff
@@ -225,7 +241,10 @@ function refreshHistoryTransactions(accountIndex: Account["accountIndex"], histo
         })
         .then((historyTransactions: Transactions) =>
           dispatch(accountDetailsActions.refreshHistoryTransactionsSuccess(historyTransactions))
-        );
+        )
+        // ToDo: Shouldn't we dispatch refreshHistoryTransactionsFailure here?
```

Comment:
I think it's better to catch and ignore and just fail silently and try again in the next refresh. No need to show anything in the UI

Reply:
Ok. Makes sense. I've removed the comment.